### PR TITLE
Rename experimental modules and packages

### PR DESCRIPTION
**Pull request by andreaTP**
A bunch of package and module renames as discussed on #579 to be able to release a first `-M1`

### CODE REVIEW

File: aot/README.md [hunk 1]
```diff
@@ -15,9 +15,14 @@ To enable use the AotMachine factory when building the module:
...
 // ...
 var is = ClassLoader.getSystemClassLoader().getResourceAsStream("compiled/basic.c.wasm");
-Instance.builder(Parser.parse(is)).withMachineFactory(AotMachine::new).build();
+Instance.
+
+builder(Parser.parse(is)).
+
+withMachineFactory(AotMachine::new).
+
+build();
```
Comment (evacchi) [inline aot/README.md:27]:
```suggestion
Instance.builder(Parser.parse(is)).withMachineFactory(AotMachine::new).build();
```

Comment (evacchi) [reply aot/README.md:27]: ...but I can't see differences, maybe it's just something to revert

**Review by evacchi — APPROVED** [review]
just one minor comment

File: aot/README.md (hunk 1 above)
Comment (andreaTP) [reply aot/README.md]: fixed

**Review by evacchi — APPROVED** [review]
re-reviewed, all good

**Pull request merged**